kernel/register.cc

```diff
@@ -576,7 +576,7 @@ void Frontend::extra_args(std::istream *&f, std::string &filename, std::vector<s
 
 	if (called_with_fp)
 		args.push_back(filename);
-	args[0] = pass_name;
+	// args[0] = pass_name;
```

Comment:
The commit message says "Comment out useless assignments." but this assignment is not useless. When called as front-end (for example via command line arguments) then `args[0]` is for example `verilog` before this assignment and `read_verilog` afterwards.

Reply:
Oops, my bad.  Shouldn't have been too hasty :).